tv_apps/smart-system/js/value_selector/value_selector.js

```diff
@@ -419,8 +419,7 @@
 
     if (this.elements.optionsTitle) {
       this.elements.optionsTitle.dataset.l10nId = titleL10nId;
-      this.elements.optionsTitle.textContent = navigator.mozL10n.get(
-        titleL10nId);
+      this.elements.optionsTitle.setAttribute('data-l10n-id', titleL10nId);
```

Comment:
you don't need those two lines here. The former can be removed in a follow up :)